src/Employer/Esfa.Recruit.Azure.Resources/azuredeploy.json

```diff
@@ -346,6 +352,109 @@
       },
       "dependsOn": []
     },
+    {
+      "apiVersion": "2017-05-10",
+      "name": "cosmos-db-database-deployment",
+      "dependsOn": [
+        "[variables('CosmosDbName')]"
+      ],
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('deploymentUrlBase'),'cosmos-db.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "cosmosDbAccountName": {
+            "value": "[variables('cosmosDbName')]"
+          },
+          "cosmosDatabase": {
+            "value": "[parameters('cosmosDatabase')]"
+          },
+          "cosmosDBType": {
+            "value": "MongoDB"
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2017-05-10",
+      "name": "[concat('cosmos-db2-container-', parameters('containerArray')[copyIndex()].containerName)]",
+      "dependsOn": [
+        "[variables('CosmosDbName')]",
+        "cosmos-db2-database-deployment"
```

Comment:
Is this supposed to be `cosmos-db-database-deployment`?

Reply:
Fixed